style.css

```diff
@@ -0,0 +1,3 @@
+.header {
+    background-color: black;
```

Comment:
olha, a cor correta é white

```suggestion
    background-color: white;
```